sklearn/svm/_base.py

```diff
@@ -862,10 +862,9 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
 
     random_state : int or RandomState instance, default=None
         The seed of the pseudo random number generator to use when shuffling
```

Comment:
same remark as discussed in another PR: it's not really a seed.

Reply:
I addressed your comment @jeremiedbb